src/RecipeCard/RecipeCard.tsx

```diff
@@ -14,6 +17,24 @@ export function RecipeCard({
   cookingTime,
   id,
 }: RecipeCardProps) {
+  const favorites = useFavoriteRecipesStore((state) => state.favorites);
+  const isFavorite = favorites.includes(id);
```

Comment:
The `.includes()` check can be done in the callback function.
```suggestion
  const isFavorite = useFavoriteRecipesStore((state) => state.favorites.includes(id));
```